app/experimenter/experiments/api/v5/types.py

```diff
@@ -191,26 +211,7 @@ def resolve_feature_value(root, info):
 class NimbusDocumentationLinkType(DjangoObjectType):
     class Meta:
         model = NimbusDocumentationLink
-        exclude = ("id", "experiment")
-
-
-class ProjectType(DjangoObjectType):
-    id = graphene.Int()
-
-    class Meta:
-        model = Project
-
```

Comment:
oh now we are adding project/team, we need this back 👀

Reply:
That should come for free with the rebase.